theme/partials/nav-item.html

```
@@ -22,92 +22,190 @@
+      {% endif %}
     </li>
+  {% else %}
+    {% if nav_item.title[0:4] == "--- " %}
+      {% if level == 2 and parent.active %}
+        {% set class = class ~ " md-nav__item--section" %}
+        {% set is_section = True %}
+      {% endif %}
+    {% endif %}
+    {% if is_section == True %}
+      <li class="{{ class }}">
+      <label class="{{ class }} md-nav__link" for="{{ path }}" id="{{ path }}_label" tabindex="{{ tabindex }}">
```

HTMLHint (1.9.2)

[error] The id value [ {{ path }}_label ] must be unique.

(id-unique)